### ISSUE TITLE
 Can this software be assigned only one algorithm task at the same time?

### ISSUE THREAD

**Issue by wuxin4692** [MoneroOcean/xmrig#102]
Can this software be assigned only one algorithm task at the same time?GPU and CPU cannot be used at the same time?

`[2022-02-16 11:23:30.444]  nvidia   use profile  cn-heavy  (2 threads) scratchpad 4096 KB
|  # | GPU |  BUS ID | INTENSITY | THREADS | BLOCKS | BF |  BS | MEMORY | NAME
|  0 |   0 | 25:00.0 |      2040 |      10 |    204 |  0 |   0 |   8160 | GeForce RTX 3080
|  1 |   1 | e2:00.0 |      2040 |      10 |    204 |  0 |   0 |   8160 | GeForce RTX 3080
[2022-02-16 11:23:30.543]  cpu      READY threads 64/64 (64) huge pages 100% 128/128 memory 262144 KB (149 ms)
[2022-02-16 11:23:31.114]  nvidia   READY threads 2/2 (669 ms)
[2022-02-16 11:23:31.396]  cpu      accepted (1/0) diff 159220 (258 ms)
[2022-02-16 11:23:33.429]  net      new job from gulf.moneroocean.stream:12048 diff 1571K algo cn-heavy/xhv height 1037793`

`[2022-02-16 10:53:48.908]  nvidia   disabled (no suitable configuration found)
[2022-02-16 10:53:48.912]  cpu      GhostRider algo 1: cn/dark (512 KB)
[2022-02-16 10:53:48.912]  cpu      GhostRider algo 2: cn/lite (1 MB)
[2022-02-16 10:53:48.912]  cpu      GhostRider algo 3: cn/fast (2 MB)
[2022-02-16 10:53:49.482]  cpu      accepted (217/0) diff 516967 (212 ms)`

**Comment by Spudz76** [comment]
Correct, when enabling more than one backend they all share the same jobs.

For separate algorithms, run separate xmrigs where only one backend type is enabled in each.
